BitriseClient/Sources/AppDelegate.swift

```diff
@@ -16,6 +16,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 
+        if ProcessInfo.processInfo.environment["TEST"] == "YESS" {
```

Comment:
`YESS` is workaround for XcodeGen's buggish behavior: https://github.com/yonaskolb/XcodeGen/issues/253
For CI we need this too. https://github.com/yonaskolb/XcodeGen/pull/254 => I'm using my fork XcodeGen for CI  https://github.com/toshi0383/XcodeGen/releases/tag/1.6.2)